### ISSUE TITLE
Error parsing message

### ISSUE THREAD

**Issue by its-me-jay** [davebryson/py-abci#24]
Hi, in the counter.py example, it is mentioned that transactions to the app are sent using "curl http://localhost:46657/broadcast_tx_commit?tx=0x01", but when I use this command, I am not getting any output. So, I changed port to 26658 ( "curl http://localhost:26658/broadcast_tx_commit?tx=0x01" ) as the ABCI server is running on port 26658. But, now I keep getting the below error on server side

![image](https://user-images.githubusercontent.com/19791181/44770632-449e6a00-ab86-11e8-97e0-3d5e6f441d82.png)

Can anyone please tell me how to figure out ths?

**Comment by joemarct** [comment]
I'm getting the same errors. Reading from the discussions in other issues, it looks like this has something to do with incompatible changes in protobuf messages in the latest version of tendermint.

**Comment by pavlovdog** [comment]
I have the same error. Tendermint version: `0.24.0-d419fffe`.
```bash
python application.py 
INFO      ABCIServer started on port: 26658
INFO      ... connection from Tendermint: 127.0.0.1:35388 ...
INFO      ... connection from Tendermint: 127.0.0.1:35390 ...
INFO      ... connection from Tendermint: 127.0.0.1:35392 ...
Traceback (most recent call last):
  File "src/gevent/greenlet.py", line 716, in gevent._greenlet.Greenlet.run
  File "/home/pavlovdog/Simple_coin/venv/lib/python3.6/site-packages/gevent/baseserver.py", line 26, in _handle_and_close_when_done
    return handle(*args_tuple)
  File "/home/pavlovdog/Simple_coin/venv/lib/python3.6/site-packages/abci/server.py", line 169, in __handle_connection
    for message in messages:
  File "/home/pavlovdog/Simple_coin/venv/lib/python3.6/site-packages/abci/encoding.py", line 59, in read_messages
    m.ParseFromString(data)
google.protobuf.message.DecodeError: Error parsing message
2018-09-14T16:05:59Z <Greenlet "Greenlet-0" at 0x7f120005a648: _handle_and_close_when_done(<bound method ABCIServer.__handle_connection of <a, <bound method StreamServer.do_close of <StreamServ, (<gevent._socket3.socket [closed]  object, fd=-1, )> failed with DecodeError
```

Here's my repo with the application code - [https://github.com/SoftblocksCo/Simple_coin](https://github.com/SoftblocksCo/Simple_coin)

**Comment by pavlovdog** [comment]
Looks like there were some breaking changes at 5 August - [link to the Tendermint CHANGELOG.md](https://github.com/tendermint/tendermint/blob/master/CHANGELOG.md#0230)

**Comment by 30mb1** [comment]
Fixed this in #26

**Comment by davebryson** [comment]
Pull request merged.  Thank you @30mb1
